### PR TITLE
Fix test failures

### PR DESCRIPTION
**Pull request by AzeemMuzammil**
# Description

$title

Fixes # (issue)

Related Pull Requests (remove if not relevant)
- Pull request 1
- Pull request 2

One line release note: 
- One line describing the feature/improvement/fix made by this PR 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Ballerina Version:
* Operating System:
* Java SDK: 

# Checklist:

### Security checks
 - [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: outlookmail/tests/test.bal [hunk 1]
```diff
@@ -428,44 +457,28 @@ function testCreateMailSearchFolder() {
...
+                } else {
+                    index = index + 1;
+                }
+            });
+        }
+        if (index == 0) {
+            break;
+        }
+        index = 0;
     }
```
Comment (NipunaRanasinghe) [inline outlookmail/tests/test.bal:483]: this seems like an infinite loop. is it intentional? shouldn't we exit once all the mails are cleared?

Comment (AzeemMuzammil) [reply outlookmail/tests/test.bal:483]: we do break with the if condition.

Comment (NipunaRanasinghe) [reply outlookmail/tests/test.bal:483]: I just noticed. my bad :)

File: outlookmail/tests/test.bal [hunk 2]
```diff
@@ -428,44 +457,28 @@ function testCreateMailSearchFolder() {
...
+        if (output is stream<Message, error?>) {
+            _ = check output.forEach(function(Message queryResult) {
+                string messageID = queryResult?.id.toString(); 
+                http:Response|error result =  outlookClient->deleteMessage(messageID, "sentitems");
+                if (result is error) {
+                    test:assertFail(msg = result.toString());
+                } else {
+                    index = index + 1;
+                }
+            });
```
Comment (NipunaRanasinghe) [inline outlookmail/tests/test.bal:477]: statements in the foreach body are not indented. Is this formatting correct?

File: outlookmail/tests/test.bal [hunk 3]
```diff
@@ -428,44 +457,28 @@ function testCreateMailSearchFolder() {
...
+        var output = outlookClient->listMessages(folderId = "sentitems", 
+            optionalUriParameters = "?$select: \"sender,subject,hasAttachments\"&top=2");
+        if (output is stream<Message, error?>) {
+            _ = check output.forEach(function(Message queryResult) {
+                string messageID = queryResult?.id.toString(); 
+                http:Response|error result =  outlookClient->deleteMessage(messageID, "sentitems");
+                if (result is error) {
+                    test:assertFail(msg = result.toString());
+                } else {
+                    index = index + 1;
```
Comment (NipunaRanasinghe) [inline outlookmail/tests/test.bal:475]: Do we need to use a counter for this purpose? can't use a boolean flag instead? that would be more clearer IMO

**Pull request closed without merging**